scripts/serve_docs

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+# Setup a temporary directory for creating a virtual environment
+temp_dir=$(mktemp -d)
+echo "Using temporary directory: $temp_dir"
+
+cleanup() {
+  echo "Cleaning up..."
+  rm -rf "$temp_dir"
+  trap - INT EXIT
+  exit
+}
+
+trap cleanup INT EXIT
+
+command_exists() {
+  type "$1" &> /dev/null;
+}
+
+# Check if uv is installed, if not, install it
+if ! command_exists uv; then
+  echo "uv is not installed. Installing uv..."
+  curl -LsSf https://astral.sh/uv/install.sh | sh
+fi
+
+# Navigate to the parent directory containing the prefect package
+cd "$(dirname "$0")/.."
+
+# Create a temporary virtual environment using uv
+uv venv "$temp_dir/.venv"
+source "$temp_dir/.venv/bin/activate"
+
+# Install the prefect package in editable mode
+uv pip install -e ".[dev]"
+
+# Install all integration libraries from src/integrations
+integration_packages=$(find src/integrations -name "pyproject.toml" -exec dirname {} \;)
+uv pip install $integration_packages
```

Comment:
It might be worthwhile to switch these two because I think the integrations packages may install a PyPI version of `prefect` that will override the editable install.

Reply:
aha! good point

Reply:
updated in [d4f7252](https://github.com/PrefectHQ/prefect/pull/12847/commits/d4f7252fc326165ecbffd43f22ba7cb509707c60)